src/pages/ProAmmPool/PositionGrid.tsx

```diff
@@ -270,4 +282,53 @@ const Row = memo(
 
 Row.displayName = 'RowItem'
 
+const emptyFunction = (): void => {
+  return
+}
+
+type DocumentPropsType = React.HTMLProps<HTMLElement>
+
+const outerElementType = forwardRef<HTMLElement, DocumentPropsType>(({ onScroll, children }, forwardedRef) => {
+  const containerRef = useRef<HTMLDivElement>(null)
+  useScroll(
+    () => {
+      if (!(onScroll instanceof Function)) {
+        return
+      }
+
+      const { clientWidth, clientHeight, scrollLeft, scrollTop, scrollHeight, scrollWidth } = document.documentElement
+
+      if (onScroll != null) {
```

Comment:
rm onScroll != null
```!(onScroll instanceof Function)``` is enough

Reply:
LOL. just copy from here https://github.com/bvaughn/react-window/issues/192. Thanks

Reply:
Updated